app/job/rest.py

```diff
@@ -150,8 +150,11 @@ def create_job(service_id):
 
     dao_create_job(job)
 
+    sender_id = data.get('sender_id')
+    sender_id = None if sender_id == 'None' else sender_id
```

Comment:
This can be removed if admin doesn't set `sender_id` on S3 metadata when it's `None`